explore/src/main/scala/explore/observationtree/ViewCommon.scala

```diff
@@ -101,13 +101,12 @@ trait ViewCommon {
     ExploreStyles.DraggingOver.when(isDragging)
 
   def undoableDeleteObs(
-    obsId:        Observation.Id,
-    observations: UndoSetter[ObservationList],
-    afterUndo:    Observation.Id => Callback,
-    afterDelete:  Callback
+    obsId:             Observation.Id,
+    observations:      UndoSetter[ObservationList],
+    @unused afterUndo: Observation.Id => Callback,
```

Comment:
This is unused but it is defined in many places it is called, I think we are missing something here

Reply:
We might have used this at some point and removed its usage later.